.github/workflows/tests.yml

```diff
@@ -0,0 +1,80 @@
+name: Test
+
+permissions:
+    contents: read
+
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    types: [opened, reopened]
```

Comment:
@donn-leaf : I found this types tidbit on the inter-webs and now this tests.yml is only run once per push instead of twice like we had been seeing all over.